tests/languages/odin/comment_feature.test

```diff
@@ -0,0 +1,90 @@
+/*
+*/
+/* A comment */
+/*!*/
+/*""*/
+/*"\a"*/
+/*#!*/
+/*' '*/
+/*'\a'*/
+/**/
+/*//*/
+/*0*/
+/*``*/
+/*`\a`*/
+/*false*/
+/*if*/
+Not a comment /* A comment */
+#!
+#! A comment
+#!!
+#!""
+#!#!
+#!' '
+#!/**/
+#!//
+#!0
+#!``
+#!false
+#!if
+Not a comment #! A comment
+//
+// A comment
+//!
+//""
+//#!
+//' '
+///**/
+////
+//0
+//``
+//false
+//if
+Not a comment // A comment
```

Comment:
Why would this not be a comment?

Reply:
I meant that the part **before** slashes is not a part of the comment. `"\r\nNot a comment ", ["comment", "// A comment"]` is the correct result.